prov/efa/src/rxr/rxr.h

```diff
@@ -175,6 +176,14 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
 
 #define RXR_MTU_MAX_LIMIT	BIT_ULL(15)
 
+/*
+ * Tentatively estimated shm space limit, 64GB
+ * If available shm space is less than this limit,
+ * shm transfer will be disabled
+ * Once shm segmentation protocol merges, we can
+ * have a better calculated limit
+ */
+#define RXR_SHM_FS_LIMIT	(68719476736UL)
```

Comment:
Nit: This could be a more readable unless you were getting C99 type warnings. `64 * 1024 * 1024 * 1024` without the suffix maybe?

Reply:
Yes, I was getting compilation warning.
```
prov/efa/src/rxr/rxr.h:186:44: warning: integer overflow in expression [-Woverflow]
 #define RXR_SHM_FS_LIMIT (64 * 1024 * 1024 * 1024)
```

Reply:
try 64LLU *1024LLU * 1024LLU *1024LLU